docs/user-guide/walkthrough/index.md

```diff
@@ -64,10 +53,12 @@ $ kubectl get pods
 
 On most providers, the pod IPs are not externally accessible. The easiest way to test that the pod is working is to create a busybox pod and exec commands on it remotely. See the [command execution documentation](/docs/user-guide/getting-into-containers/) for details.
 
-Provided the pod IP is accessible, you should be able to access its http endpoint with curl on port 80:
+Provided the pod IP is accessible, you should be able to access its http endpoint with wget on port 80:
 
 ```shell
-$ curl http://$(kubectl get pod nginx -o go-template={{.status.podIP}})
+$ kubectl run busybox --image=busybox --restart=Never --tty -i  --env "POD_IP=$(kubectl get pod nginx -o go-template={{.status.podIP}})"
```

Comment:
`{{.status.podIP}}` doesn't seem to be escaped when I hit view.  If I copy and paste the commands, they produce the desired result.  (getting the nginx page)